server/graphql/v2/interface/Account.js

```diff
@@ -224,6 +224,19 @@ const accountFieldsDefinition = () => ({
       return collective;
     },
   },
+  hasTwoFactorAuth: {
+    type: GraphQLBoolean,
+    async resolve(collective) {
+      const user = await models.User.findOne({
+        where: { CollectiveId: collective.id },
+      });
+      if (user.twoFactorAuthToken) {
```

Comment:
Same here, this will crash if we try to fetch this field on something else than a user collective. For example with an org, `models.User.findOne` will return `null`